src/part-progress/part-progress.swagger.ts

```diff
@@ -0,0 +1,44 @@
+import { applyDecorators } from '@nestjs/common';
+import { ApiBody, ApiOperation, ApiResponse } from '@nestjs/swagger';
+import { ResPartProgressDto } from './dto/res-part-progress.dto';
+import { CreatePartProgressDto } from './dto/create-part-progress.dto';
+
+export const ApiPartProgress = {
+  findAll: () => {
+    return applyDecorators(
+      ApiOperation({
+        summary: '유저의 전체 문제 풀이 현황 조회',
+      }),
+      ApiResponse({
+        status: 200,
+        description: `
+        유저와 파트 사이의 진행도를 모두 보여줌
+        `,
+        type: ResPartProgressDto,
+        isArray: true,
+      }),
+    );
+  },
+  createOrUpdate: () => {
+    return applyDecorators(
+      ApiOperation({
+        summary: '유저의 파트에 대한 진행도생성 또는 업데이트',
+      }),
+      ApiBody({
+        description: `
+        part의 status 값 넣기
+        1. LOCKED
+        2. STARTED
+        3. IN_PROGRESS
+        4. COMPLETED
+        `,
+        type: CreatePartProgressDto,
+      }),
+      ApiResponse({
```

Comment:
```suggestion
      ApiNoContentResponse({
```
위 데코레이터를 쓰면 status 값을 생략할 수도 있긴 합니다.
편한데로 하시면 될 것 같아요